src/abstraction/nv.rs

```diff
@@ -37,3 +39,25 @@ pub fn read_full(
 
     Ok(result)
 }
+
+pub fn list(context: &mut Context) -> Result<Vec<(NvPublic, Name)>> {
+    context.execute_without_session(|context| {
+        let (handles, _) = context.get_capabilities(
+            CapabilityType::Handles,
+            u32::to_be(TPM2_HT_NV_INDEX.into()),
```

Comment:
Just a question, didn't you already create a ```PropertyTag``` type? So why is not used in ```get_capabilities```?

Reply:
Because `PropertyTag` is tags for `CapabilityType::TPMProperties`, i.e. for TPM Properties (with thoughts of also including `CapabilityType::PCRProperties`). This is `CapabilityTypes::Handles`.

Reply:
Ahh I see so that ```property``` variable is the 'Group' i.e. the type of handle (transient, persistent, permanent, nv_index ...).

Reply:
Almost. The first argument is `capability`, which is the group. Then the second one is `property`, which is which property to get (or start at), and then there's `propertyCount` to indicate how many.
e.g. `(TPM_Properties, VENDOR_STRING_1, 3)` would return `VENDOR_STRING_1`, `VENDOR_STRING_2, and `VENDOR_STRING_3`, but not `VENDOR_STRING_4`.

Reply:
Yeah, that was what I meant. One thing though should it not be ```TPM2_NV_INDEX_FIRST``` for correctness? Not that it matters because in this case they have the same value.

But the specification says:
>The capability parameter determines the category of data returned. The property parameter selects the
>first value of the selected category to be returned. If there is no property that corresponds to the value of
>property, the next higher value is returned, if it exists.

Reply:
Yeah, fair enough. Updated that.